OutOfSchool/OutOfSchool.ElasticsearchData/Models/WorkshopES.cs

```diff
@@ -24,6 +25,8 @@ public class WorkshopES
 
     public string ProviderTitle { get; set; }
 
+    public ProviderStatus ProviderStatus { get; set; }
```

Comment:
i don't see any logic the allows you to update this field when the provider status changes for provider in database.

Unless you make a force update to the workshop in any way in database using workshop combiner methods, any updates to provider will not be synchronised to Elastic and this field will not hold up to date information.